integrationTests/vm/arwen/testdata/upgrades-parent/parent.cpp

```diff
@@ -0,0 +1,94 @@
+typedef unsigned char byte;
+typedef unsigned int u32;
+typedef unsigned long long u64;
+
+extern "C" void int64finish(long long value);
```

Comment:
It's cleaner to use an `extern "C" { ... }` block instead of prefixing each API method with `extern "C"`. But do leave the SC methods with individual `extern`s.

Reply:
Fixed.